src/main/java/service/strategy/strategyimpl/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package service.strategy.strategyimpl;
+
+import dao.Storage;
+import model.FruitRecord;
+
+public class BalanceOperation  implements OperationHandler{
+    @Override
+    public void apply(FruitRecord transaction) {
+        Storage.storage.put(transaction.getFruit(), transaction.getQuantity());
+    }
```

Comment:
It's good to see that you have implemented the Strategy pattern by creating a `BalanceOperation` class that implements the `OperationHandler` interface. However, please ensure that the `BalanceOperation` class is placed in the correct package according to the project structure. According to the checklist, the `strategy` package should contain the `OperationHandler` interface and its implementations, but it seems like you have an additional `strategyimpl` package which is not necessary. Consider restructuring your packages to align with the checklist requirements.